exploration/number-selection.md

```diff
@@ -45,6 +47,7 @@ but ordinal rules use `one` (_1st_, _21st_, etc.), `few` (_2nd_, _22nd_, etc.),
 Additionally,
 MF1 provides `ChoiceFormat` selection based on a complex rule set
 (and which allows determining if a number falls into a specific range).
+This capability is deprecated.
```

Comment:
As I understand it, this is already effectively deprecated (but technically supported) in MF1, so being clearer that we're not doing it at all might be better. Though it's clearly still somewhat present in our design, as e.g. the MatchSelectKeys function interface could not be simplified because someone might want to implement something like ChoiceFormat in MF2.

```suggestion
This capability is not supported by the default functions of MF2.
```